Formula/opencv@3.rb

```diff
@@ -83,26 +99,24 @@ def install
       -DWITH_VTK=OFF
       -DBUILD_opencv_python2=OFF
       -DBUILD_opencv_python3=ON
-      -DPYTHON3_EXECUTABLE=#{Formula["python@3.9"].opt_bin}/python3
+      -DPYTHON3_EXECUTABLE=#{which("python3")}
     ]
 
-    if Hardware::CPU.intel?
+    if Hardware::CPU.intel? && build.bottle?
       args << "-DENABLE_AVX=OFF" << "-DENABLE_AVX2=OFF"
       args << "-DENABLE_SSE41=OFF" << "-DENABLE_SSE42=OFF" unless MacOS.version.requires_sse42?
     end
 
-    mkdir "build" do
-      system "cmake", "..", *args
-      inreplace "modules/core/version_string.inc", Superenv.shims_path, ""
-      system "make"
-      system "make", "install"
-      system "make", "clean"
-      system "cmake", "..", "-DBUILD_SHARED_LIBS=OFF", *args
-      inreplace "modules/core/version_string.inc", Superenv.shims_path, ""
-      system "make"
-      lib.install Dir["lib/*.a"]
-      lib.install Dir["3rdparty/**/*.a"]
-    end
+    system "cmake", "-S", ".", "-B", "build_shared", *args
+    inreplace "build_shared/modules/core/version_string.inc", "#{Superenv.shims_path}/", ""
+    system "cmake", "--build", "build_shared"
+    system "cmake", "--install", "build_shared"
+
+    system "cmake", "-S", ".", "-B", "build_static", *args, "-DBUILD_SHARED_LIBS=OFF"
+    inreplace "build_static/modules/core/version_string.inc", "#{Superenv.shims_path}/", ""
+    system "cmake", "--build", "build_static"
+    lib.install Dir["build_static/lib/*.a"]
+    lib.install Dir["build_static/3rdparty/**/*.a"]
```

Comment:
Installing third-party static libraries is really suspicious.

Reply:
There are some Intel libraries (ippicv, ittnotify), quirc, etc. that get built. Not sure if these are needed for main static libraries to work. I just removed the 3rd party libraries that we had alternative formulae for.